.github/workflows/pr.yml

```diff
@@ -26,3 +26,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: seL4/ci-actions/bashisms@master
+
+  style:
+    name: Style
+    runs-on: ubuntu-latest
+    steps:
+    - uses: seL4/ci-actions/style@master
```

Comment:
Could this use local paths now?

Reply:
Seems this does not work as relative paths are resolved against the caller's repo. :(

Reply:
yes, correct